page_objects/examples/weather_shopper_mobile_app/weather_shopper_cart_page.py

```diff
@@ -0,0 +1,70 @@
+"""
+Page objects for cart page in Weathershopper application.
+"""
+# pylint: disable = W0212,E0401
+import re
+import conf.locators_conf as locators
+from utils.Wrapit import Wrapit
+from core_helpers.mobile_app_helper import Mobile_App_Helper
+
+class WeatherShopperCartPage(Mobile_App_Helper):
+    "Page object for the cart page in Weathershopper application."
+
+    @Wrapit._exceptionHandler
+    @Wrapit._screenshot
+    def get_cart_total(self):
+        "This method is to get the total price in the cart."
+
```

Comment:
Maintain formatting in all pages... refer app_helpers and selenium tutorial page objects

Reply:
Fixed the formatting and added conditional writes